corehq/blobs/migratingdb.py

```diff
@@ -45,3 +45,7 @@ def get_path(self, *args, **kw):
 
     def copy_blob(self, content, info, bucket):
         self.new_db.copy_blob(content, info, bucket)
+
+    def close(self):
```

Comment:
Thanks. I think the correct setup is now used https://github.com/dimagi/commcare-hq/pull/16808/commits/969fe76a65d396fa98d0a9796b1a71c5891324ce